src/System.Management.Automation/engine/Interop/Windows/GetCurrentThreadId.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+#if !UNIX
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Windows
+    {
+        [LibraryImport("api-ms-win-core-processthreads-l1-1-0.dll")]
```

Comment:
Why not use PinvokeDllNames.GetCurrentThreadIdDllName here?

Reply:
The name is used only once. 
Most of names from PinvokeDllNames are used 1 or 2 times. It seems it makes no sense to use named consts. They make reading more difficult than helpful.